chainer/functions/utils/forget.py

```diff
@@ -59,4 +59,59 @@ def backward(self, inputs, grads):
 
 
 def forget(func, *xs):
+    """Call a function without storing internal results.
+
+    On a forward-propagation Chainer stores all internal results of
```

Comment:
"forward-propagation" It seems that "forward propagation" (without hyphen) is used many times in this documentation. The same thing holds for the notations below.